google/cloud/bigquery/client.py

```diff
@@ -1935,9 +1935,8 @@ def create_job(
                 source_type=source_type,
             )
         elif "query" in job_config:
-            copy_config = copy.deepcopy(job_config)
             query_job_config = google.cloud.bigquery.job.QueryJobConfig.from_api_repr(
-                copy_config
+                copy.deepcopy(job_config)
```

Comment:
Are we making a copy because we expect `from_api_repr` to mutate the dict?

Reply:
No, I made a copy because I didn't pay close enough attention to your suggestion. :)

Fixed.